app/code/Magento/Checkout/view/frontend/web/js/action/select-shipping-address.js

```diff
@@ -12,6 +11,8 @@ define([
     'use strict';
 
     return function (shippingAddress) {
+        var address = quote.billingAddress();
         quote.shippingAddress(shippingAddress);
+        quote.billingAddress(address);
```

Comment:
Hi @Dharmeshvaja91. Could you please explain why you set the same value to the same observable variable. How it fix the issue?